### ISSUE TITLE
Storyboard execution should continue until stop trigger

### ISSUE THREAD

**Issue by aG-MathWorks** [esmini/esmini#302]
esmini ends simulation when all acts are complete (with message "All acts are done, quite now"). However, the OSCv1.1 and 1.2 user guides state that execution should continue until the storyboard stop trigger is satisfied:

![image](https://user-images.githubusercontent.com/87451108/178333973-6ae8c123-33db-4aa8-ae34-dadacf620d4c.png)

We can work around this issue by generating a dummy/placeholder Act with no stop trigger, but it would be ideal if esmini adhered to the simulation execution guidelines here.

Would a fix for this be welcome, or is there some motivation for the discrepancy? 

Thanks for your continued support of this great tool!

**Comment by eknabevcc** [comment]
Yes, you're right. The scenario story should not stop on the completion of its acts. And yes, a fix for this would be welcome (e.g. by a PR). 👍

BTW, since [OSC 1.0](https://www.asam.net/index.php?eID=dumpFile&t=f&f=3496&token=df4fdaf41a8463e585495001cc3db3298b57d426) suggests that the Story is done when all associated acts are done ("A Story's goal is accomplished when all its Acts are in the completeState."), I guess it would be nice to keep current behavior for 1.0 - if not too complicated... Or do you have another interpretation/opinion?
